app/assets/stylesheets/submission.css

```diff
@@ -13,6 +13,11 @@ table.results {
   vertical-align: top;
 }
 
+.results .inspect_output th {
+  padding: 0;
```

Comment:
The change breaks the alignment with "Wall time" from the previous row.
Before:
![1-before](https://user-images.githubusercontent.com/5887562/75104349-098ed100-566d-11ea-98f6-ceddd57b582a.png)
After:
![2-after](https://user-images.githubusercontent.com/5887562/75104347-04ca1d00-566d-11ea-81ab-b2ce3c68dd9b.png)
It would be nice if they remained aligned, but I don't have a good way of doing that (yet).

Reply:
I don't think this is a major concern since the wall time information isn't available to standard users. But yes, if there is an easy fix it would be nice

Reply:
Also note that the Wall time is already slightly misaligned in the original version (due to `border-spacing: 5px;`)

Reply:
How about?
<img width="850" alt=" 2020-02-24 at 12 07 39 AM" src="https://user-images.githubusercontent.com/12654833/75110996-c860e600-5699-11ea-8bbf-2122d81b8684.png">